README.md

```diff
@@ -12,6 +12,7 @@ The purpose of this project is to make time series manipulation with Spark simpl
 [![codecov](https://codecov.io/gh/databrickslabs/tempo/branch/master/graph/badge.svg)](https://codecov.io/gh/databrickslabs/tempo)
 [![Downloads](https://pepy.tech/badge/dbl-tempo/month)](https://pepy.tech/project/dbl-tempo)
 [![PyPI version](https://badge.fury.io/py/dbl-tempo.svg)](https://badge.fury.io/py/dbl-tempo)
+[![docs](https://img.shields.io/badge/docs-ghpages-lemon)](https://tempo.github.io)
 
```

Comment:
This is awesome - I think the hyperlink is great - in terms of the text shown. Instead of ghpages, I wonder if we can display a status of the docs (e.g. passing instead of `ghpages`. Here is an example from dbx - .. |docs| image:: https://readthedocs.org/projects/dbx/badge/?version=latest
    :target: https://dbx.readthedocs.io/en/latest/?badge=latest
    :alt: Documentation Status)

Reply:
@rportilla-databricks 
Yeup makes sense to have a _docs deployment status_ instead of the text `docs | ghpages`. For DBX its easy to do as its hosted on readthedocs. 

But I got the point Ricardo, let me check with @tnixon on Monday, If we could have **_a separate workflow only for the sphinx deployment on github pages_** then we could have that _**docs workflows status badge**_ like we have for the build status <img width="94" alt="image" src="https://user-images.githubusercontent.com/98817232/159140284-a5311c3e-5a5b-4aab-9c34-3d3a61694193.png">

Reply:
@rportilla-databricks 

I have made the changes now we have the `docs | passing` badge
<img width="873" alt="image" src="https://user-images.githubusercontent.com/98817232/159570918-ac2b6891-0f4d-4b9a-935d-3183eb70b30f.png">

With the badge hyperlinked to the correct docs page also: 
<img width="239" alt="image" src="https://user-images.githubusercontent.com/98817232/159571069-cf2967c9-212a-4277-8815-05c6fd2b9272.png">